AnnoyancesFilter/sections/popups.txt

```diff
@@ -123,6 +123,9 @@ theoryandpractice.ru##a[href^="https://form.typeform.com/"]
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt #########
 !
+esprit.de#$#.mfp-wrap { display: none !important; }
+esprit.de#$#.mfp-bg { display: none !important; }
+esprit.de#$#html { overflow: auto !important; padding-right: 0 !important; }
```

Comment:
How to reproduce it? Please send a screenshot with code of the element.

Reply:
@zloyden Well, I can't reproduce this time.

Reply:
Please remove these rules except your fix for `spreadshirt`.